packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistoryTable.tsx

```diff
@@ -256,13 +256,14 @@ export const TransactionHistoryTable = ({
             )}
             className="table-auto last:border-b-0"
             rowGetter={({ index }) => transactions[index]}
-            rowRenderer={({ index, key, style }) => {
+            rowRenderer={({ index, style }) => {
               const tx = transactions[index]
 
               if (!tx) {
                 return null
               }
 
+              const key = `${tx.parentChainId}-${tx.childChainId}-${tx.txId}`
```

Comment:
If row unmounts, e.g. completed tx, the key is not correct (probably is based on index), and causes side effects.